### ISSUE TITLE
how to process the original datasets to match the DataLoader?

### ISSUE THREAD

**Issue by WooXinyi** [chufengt/iccv19_attribute#8]
What is the train_list_path file in datasets.py? I follow the given example but only got .pkl file.

**Comment by chufengt** [comment]
similar to #4

**Comment by WooXinyi** [comment]
> similar to #4

thanks, i have made it in that way. Besides, where can i got the bn_inception-52deb4733.pth? Or it is not necessary to use a pre-trained model?

**Comment by syusukee** [comment]
@WooXinyi Have you succeed in transforming data, could you share how to do it?Thanks a lot!!

**Comment by WooXinyi** [comment]
> @WooXinyi Have you succeed in transforming data, could you share how to do it?Thanks a lot!!

import scipy.io as sio
import numpy as np


train_file = '../dataset/peta/train.txt'
val_file = '../dataset/peta/val.txt'

lines = sio.loadmat('../PETA.mat')['peta'][0][0][0]
with open(train_file, 'w') as f1:
....with open(val_file, 'w') as f2:
........for line in lines[:18000]:
............f1.write((str('%05d'%line[0])+'.png'+' '+str(line[4:39])[1:-1]).replace('\n', '')+'\n')
........for line in lines[18000:]:
............f2.write((str('%05d' % line[0]) + '.png' + ' ' + str(line[4:39])[1:-1]).replace('\n', '') + '\n')

Hope be helpful.

**Comment by syusukee** [comment]
@WooXinyi It really helps me a lot!! Thank you so much!

**Comment by happyprotean** [comment]
@WooXinyi Hi, The author choose 35 attribute which positive ratio is higher than 5% in the paper, but you just choose first 35 attributes in your code. I found not all first 35 attributes' positive ratio are higher than 5%, some just 2%.
@chufengt  Can you provide the .txt file which is used in your training? I want to keep exact same setting with you to reproduce the result. Otherwise the result is not so convincing. Thanks a lot!

**Comment by WooXinyi** [comment]
> @WooXinyi Hi, The author choose 35 attribute which positive ratio is higher than 5% in the paper, but you just choose first 35 attributes in your code. I found not all first 35 attributes' positive ratio are higher than 5%, some just 2%.
> @chufengt Can you provide the .txt file which is used in your training? I want to keep exact same setting with you to reproduce the result. Otherwise the result is not so convincing. Thanks a lot!

but the attributes I used are the same with author's

**Comment by happyprotean** [comment]
Yes, you have used the same attributes with author's attributes in the code (dataset.py). But the question is whether these attribute are same with paper's?

**Comment by WooXinyi** [comment]
> Yes, you have used the same attributes with author's attributes in the code (dataset.py). But the question is whether these attribute are same with paper's?

Sorry I didn't consider it, maybe you could find those attributes which positive ratio are higher than 5%.

**Comment by chufengt** [comment]
> @WooXinyi Hi, The author choose 35 attribute which positive ratio is higher than 5% in the paper, but you just choose first 35 attributes in your code. I found not all first 35 attributes' positive ratio are higher than 5%, some just 2%.
> @chufengt Can you provide the .txt file which is used in your training? I want to keep exact same setting with you to reproduce the result. Otherwise the result is not so convincing. Thanks a lot!

I just download the label list from https://github.com/asc-kit/vespa/tree/master/generated .

**Comment by chufengt** [comment]
I've uploaded the label lists.
